public/app/features/manage-dashboards/components/PublicDashboardListTable/PublicDashboardListTable.tsx

```diff
@@ -41,17 +43,30 @@ export const PublicDashboardListTable = () => {
             </tr>
           </thead>
           <tbody>
-            {publicDashboards?.map((pd) => (
+            {publicDashboards?.map((pd: ListPublicDashboardResponse) => (
```

Comment:
Nice!

Reply:
yeah.. actually, this should be inferred but for some reason, it's not working. gonna dive into it later